js/index.js

```diff
@@ -46,3 +46,81 @@ for(let i=0 ;i< skills.length ; i++ )
       skillsList.appendChild(skill);
     }
 
+    /*Create a variable named messageForm that uses "DOM Selection" to select the 
+    "leave_message" form by name attribute*/
+
+    const messageForm = document.querySelector('form');
+
+    /*Add an event listener to the messageForm element that handles the "submit" event
+    Inside the callback function for your event listener, create three new variables 
+    (one for each of the three form fields) and retrieve the value from the event*/
+    
+    messageForm.addEventListener('submit',event=>{
+        event.preventDefault();                  
+        const username = event.target.usersName.value;
+        const email= event.target.usersEmail.value;
+        const usermessage=event.target.usersMessage.value;
+        console.log("username is ", username);
+        console.log("Email is ", email);
+        console.log("user Message is ", usermessage);
+        
+        /*Create a variable named messageSection and use "DOM Selection" to select the 
+        #messages section by id*/
+        const messageSection=document.getElementById('messages');
+
+        /*Create a variable named messageList and use "DOM Selection"
+         to query the messageSection (instead of the entire document) to find the <ul> element*/
+         const messageList = messageSection.querySelector('ul');
+
+         /*Create a variable named newMessage that makes a new list item (li) element*/
+         const newMessage = document.createElement('li');
+         
+
+         /*On the next line, set the inner HTML of your newMessage element with the following 
+         information:<a> element that displays the "usersName" and is a clickable link to the 
+         "usersEmail" (hint: use the mailto: prefix) <span> element that displays the "usersMessage"*/
+         const alink =document.createElement('a');
+         alink.innerHTML=`${username} `;
+         alink.href=`mail to: https://${email} `;
```

Comment:
Nice work with selecting the correct parts of the form throughout this function. 

I just noticed one thing here. When this link is clicked on, we're wanting the user's email client to open a new email with the address populated. Have a look [here](https://www.freecodecamp.org/news/mailto-link-how-to-make-an-html-email-link-example-code/) on how to format that link.